tests/ManageCourses.Tests/TestUtilities/ApiProcessLauncher.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using System.Diagnostics;
+using System.Threading;
+using Microsoft.Extensions.Configuration;
+
+namespace GovUk.Education.ManageCourses.Tests.TestUtilities
+{
+    public class ApiProcessLauncher
+    {
+        public DisposableProcess LaunchApiLocally(IConfiguration config)
```

Comment:
OneTimeSetup
var site = WebHost.CreateDefaultBuilder(args)
                .UseStartup<Startup>()

Setup
site.Start()

Teardown
site.Stop()

Reply:
Clever! I got this to work with some wrestling